homework/oop/homework_3.py

```diff
@@ -0,0 +1,27 @@
+# Home task 3
+
+class BaseClass1:
+    def test(self):
```

Comment:
Namespace 'test' is usually reserved in the test automation frameworks and in most cases you should avoid using it. Or constantly keep this fact in mind